### PR TITLE
New Corporate Coats

### PR DESCRIPTION
**Pull request by 1Sparky1**
Adds new corporate variants of the departmental jackets and winter coats. Also created a NanoTrasen and Orion coat selection in the loadout, with these and their labcoats.

Fixed a few missed cargo references in the loadout.

### CODE REVIEW

File: code/modules/clothing/suits/storage.dm [hunk 1]
```diff
@@ -91,3 +91,63 @@
...
+
+// Corporate Jackets
+
+/obj/item/clothing/suit/storage/toggle/corp/idris
+	name = "idris corporate jacket"
+	desc = "A cozy jacket in Idris' colors, for those who need more merchandise in their life."
+	icon_state = "idris_corp_jacket"
+
+/obj/item/clothing/suit/storage/toggle/corp/idris/alt
+	icon_state "idris_corp_jacket_alt"
```
Comment (alsoandanswer) [inline code/modules/clothing/suits/storage.dm:103]:
```suggestion
	icon_state = "idris_corp_jacket_alt"
```

**Comment by Greenjoe12345** [comment]
Could you add the Orion beret to the loadout, too? I noticed it's a thing in the code but not in the loadout.

**Comment by 1Sparky1** [comment]
> Could you add the Orion beret to the loadout, too? I noticed it's a thing in the code but not in the loadout.

I was probably going to add other Orion drip in another PR. I happened to be on a coat spree so I did those here.

**Review by alsoandanswer — CHANGES_REQUESTED** [review]
Right, some issues.

Stray pixels on zeng_corp_jacket_open, west-facing.

epmc_corp_jacket, zeng_corp_jacket_alt, orion_corp_jacket all have off-palette colors.
Please fix this, thanks. A good way to check (in dreammaker) is using the colors in use section on the right.

File: code/modules/client/preference_setup/loadout/loadout_factions.dm [hunk 2]
```diff
@@ -488,17 +521,48 @@
 	faction = null
 
 /datum/gear/faction/scc_jacket
-	display_name = "SCC jacket"
+	display_name = "SCC coat selection"
+	description = "A seclection of SCC coats"
```
Comment (GMR25) [inline code/modules/client/preference_setup/loadout/loadout_factions.dm:525]: Typo on selection.

**Review by alsoandanswer — APPROVED** [review]
Checks :
Winter Coats : All good. Unfortunate the "hood fluff" isn't part of the palette, but it's fine.
Department Jackets : All good.
Labcoats : All good.

You've done some absolutely stellar work here.
Apologies for the delay, it's been a busy month.

**Comment by alsoandanswer** [comment]
!review

**Review by Alberyk — APPROVED** [review]

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]
contained sprite paths.
don't add sprites to `icons/obj/clothing/suits.dmi` nor `icons/mob/suit.dmi`.

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]
corp jackets are still in the last review's mentioned DMI files.

**Review by SleepyGemmy — APPROVED** [review]

**Pull request merged**